### ISSUE TITLE
Add Activation Functions to Paddle Backend

### ISSUE THREAD

**Issue by MahmoudAshraf97** [unifyai/ivy#10649]
Add [Activation Functions](https://www.paddlepaddle.org.cn/documentation/docs/en/api/index_en.html) to Paddle backend
Please fork from the [`PaddlePaddle branch`](/unifyai/ivy/tree/PaddlePaddle) instead of `master`

- [x] relu
- [x] leaky_relu
- [x] gelu
- [x] sigmoid
- [x] softmax
- [x] softplus
- [x] log_softmax
- [x] mish
## Experimental
- [x] logit
- [x] thresholded_relu
- [x] relu6

**Comment by asadimtiazmalik** [comment]
Hi I want to work on one of these issues but I have 2 primary questions:

1.  I created a pull-request which is under-review and that issue is still on going, so can I work on one of these.
2. I can't find paddle directory in the backend folder so in case i contribute where should i write the code

**Comment by MahmoudAshraf97** [comment]
Hi @asadimtiazmalik , you can work on the PaddlePaddle branch for now until we merge with the master branch
